source/query-with-natural-language/query-with-natural-language.txt

```diff
@@ -0,0 +1,72 @@
+.. _query-natural-language:
+
+===========================
+Query with Natural Language
+===========================
+
```

Comment:
I also just realized that we're not calling out the fact that this feature is in public preview anywhere. We should at least call it out on this page but perhaps on the task pages as well? Happy to follow whatever paradigm we do in our other docs

Reply:
I added a blurb to the banner in order to reduce the amount of addmonitions on the pages, I copied the text from the public preview notice we had on the VSCode extension docs:


```
The MongoDB Compass natural language querying feature is currently 
available as a **Public Preview** offering. The product, its features, 
and the corresponding documentation may change during the Preview stage.
 ```
 
<img width="825" alt="Screenshot 2023-09-21 at 8 59 43 AM" src="https://github.com/mongodb/docs-compass/assets/85948430/5e2498e7-0083-4c88-94c6-4707e445b66b">